pages/syllabus.md

```diff
@@ -7,65 +7,116 @@ description: Syllabus for DSCI 345
 See [the schedule](schedule.html) for a list of topics by week, with links to slides.
 
 
-# Course and Instructor Information
-- Math 345 CRN 11689, Stats for Data Science
-- Dsci 345 CRN 13329, Stats for Data Science
+# Instructors:
 
-Instructor: Benjamin Young, Fenton 211, bjy@uoregon.edu.  Pronouns: He/They.
+- [Peter Ralph](https://kr-colab.github.io/people), instructor: plr@uoregon.edu
+    * lecture: 11-11:50 MWF in ESL 105
+    * office hours: TBD
 
-This is a slightly summarized version of the syllabus - I've taken out any links that I didn't want to post on the public internet.  Official version is on the Canvas site.
-# Office Hours
+- [Ben Young](https://pages.uoregon.edu/bjy/), instructor: bjy@uoregon.edu
+    * lecture: 11-11:50 MWF in LA 230
+    * office hours: TBD
 
-- 10am-11am Tuesdays and Fridays, or by appointment; please see Canvas for my appointment booking link.  
 # Course Description
 
-This course covers a theoretical basis in probability and statistics that is foundational for work in data science. Randomness and uncertainty are fundamental at all stages of data science work, from understanding strengths and limitations of data sources, through modeling and inference, to algorithmic aspects of learning and prediction (e.g., random forests; stochastic gradient decent). This course is aimed at students who have done some work looking at real data and want to have a deeper (and more quantitative) understanding of what we are doing when we make predictions using data. The course covers both tools for modeling randomness and calculating properties of those models, and the process of estimating quantities from data. An important thread throughout the course is on _simulating data_: being able to construct and simulate from sophisticated models for random data generation.
```

Comment:
I've inserted semantic linebreaks since - like in latex - they don't affect the downstream display and I find it easier to read and edit.